bin/pdc_client

```diff
@@ -151,6 +151,8 @@ if __name__ == "__main__":
                                         "debugging.",
                         action="store_true", default=False)
     parser.add_argument("-c", "--comment", help="Reasons for the PDC change.")
+    parser.add_argument('--version', action='version',
+                        version='%(prog)s ' + pdc_client.__version__)
```

Comment:
This  pdc_client.__version__ is call the __version__ in __init__.py, right?
Maybe remove the line 22 as it has no use?

Reply:
Cool, that's also my idea!